core/impl/src/main/java/com/blazebit/persistence/impl/function/datediff/millisecond/OracleMillisecondDiffFunction.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2014 - 2017 Blazebit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blazebit.persistence.impl.function.datediff.millisecond;
+
+import com.blazebit.persistence.spi.FunctionRenderContext;
+import com.blazebit.persistence.spi.TemplateRenderer;
+
+/**
+ *
+ * @author Christian Beikov
+ * @since 1.0
+ */
+public class OracleMillisecondDiffFunction extends MillisecondDiffFunction {
+
+    private final TemplateRenderer bothParamRenderer;
+    private final TemplateRenderer firstParamRenderer;
+    private final TemplateRenderer secondParamRenderer;
+
+    public OracleMillisecondDiffFunction() {
+        super("1000 * (extract(second from cast(?2 as timestamp) - cast(?1 as timestamp))\n" +
```

Comment:
Remove all the `\n` please and maybe also add the source: http://stackoverflow.com/questions/22606161/difference-between-timestamps-in-milliseconds-in-oracle